distribution/scripts/jmeter/jtl-splitter.sh

```diff
@@ -53,5 +53,7 @@ if [[ -z $heap_size ]]; then
     echo "Please specify the heap size."
     exit 1
 fi
-
+var_path=`pwd`
```

Comment:
The same question here. What is the directory path (value of `var_path`) we are at when generating the payloads?